Dependencies/xr/Source/OpenXR/include/IXrContext.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <openxr/openxr.h>
+#include <openxr/openxr_platform.h>
+#include <XrUtility/XrExtensions.h>
+
+#if defined(__CYGWIN32__)
+    #define OPENXR_CONTEXT_INTERFACE_API __stdcall
+#elif defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(_WIN64) || defined(WINAPI_FAMILY)
+    #define OPENXR_CONTEXT_INTERFACE_API __stdcall
+#elif defined(__MACH__) || defined(__ANDROID__) || defined(__linux__) || defined(LUMIN)
+    #define OPENXR_CONTEXT_INTERFACE_API
+#else
+    #define OPENXR_CONTEXT_INTERFACE_API
+#endif
+
+typedef struct IXrContext

```

Comment:
>IXrContext [](start = 15, length = 10)

Discussed offline, let's disambiguate this name to be OpenXR-specific

Reply:
sounds good